README.md

```diff
@@ -22,15 +30,14 @@ To run the test you need to next data (you can see an example of them at the `ex
 
 ### Configuration
 
-If you add a file names ```.lsit.yml``` into your main logstash directory
-you can have your configuration and input files in non standard
+If you add a file named ```.lsit.yml``` in your main logstash directory you can have your configuration and input files in non standard
```

Comment:
Well, nowadays the file need to be named like this. This change will need some changes in the gem. I think we can do it here.